### PR TITLE
Removed badges and tech stack if the user is not dev

### PR DESCRIPTION
**Pull request by saishankarmanugula**
Changes related to issue 168.
closes 168

image

image

### CODE REVIEW

File: pages/[id]/index.js [hunk 1]
```diff
@@ -14,9 +15,17 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
   const { first_name = '', last_name = '' } = user;
   const memberName = `${first_name} ${last_name} | Member Real Dev Squad`;
 
+  const router = useRouter();
+  const devUser = router.query.dev ? true : false;
```
Comment (sumitd94) [inline pages/[id]/index.js:19]: We can make it look more readable using double negation
```javascript
const devUser = !!query.dev
```

File: pages/[id]/index.js [hunk 2]
```diff
@@ -14,9 +15,17 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
   const { first_name = '', last_name = '' } = user;
   const memberName = `${first_name} ${last_name} | Member Real Dev Squad`;
 
+  const router = useRouter();
```
Comment (sumitd94) [inline pages/[id]/index.js:18]: can we destructure `query` here?
```javascript
const { query } = useRouter();
```

**Review by sumitd94 — CHANGES_REQUESTED** [review]
One small change requested Please, Rest all looks good 🥇

File: pages/[id]/index.js (hunk 1 above)
Comment (saishankarmanugula) [reply pages/[id]/index.js]: done

File: pages/[id]/index.js (hunk 2 above)
Comment (saishankarmanugula) [reply pages/[id]/index.js]: done

**Review by sumitd94 — APPROVED** [review]
Great Job done 🥇

**Pull request merged**